io.github.eteran.edb-debugger.yml

```diff
@@ -0,0 +1,78 @@
+app-id: io.github.eteran.edb-debugger
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: edb
+rename-icon: edb
+rename-desktop-file: edb.desktop
+rename-appdata-file: edb.appdata.xml
+finish-args:
+  - --allow=devel
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=xdg-documents
+  - --device=dri
+
+modules:
+  - name: capstone
+    buildsystem: simple
+    build-commands:
+      - make V=1 PREFIX=/app/
+      - make DESTDIR=/app install
```

Comment:
`DESTDIR` doesn't do what you think it does. This should be `PREFIX` and then you can likely remove the following lines, including the `post-install`

Reply:
the usr directory becomes empty and causes an error in my local tests.

Reply:
because there is no `/usr` directory as the prefix is `/app`.

Reply:
without DESTDIR the installation is attempted in /usr which causes an error.

Reply:
I will repeat one more time: it's because you need to set prefix instead.